src/components/browseButton/index.js

```diff
@@ -48,13 +49,15 @@ export function setup ({
     displayCurrentPrimaryAnnotations,
     getContentFiles,
     getAnnoFiles,
-    closePDFViewer
+    closePDFViewer,
+    restoreBeforeStatus
```

Comment:
* How about this name?
```
restoreBeforeState -> callbackLoadedFiles
```
I understood your situation, and it may be better to change the name more abstract for common.